classes/class-p4-taxonomy-page.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+if ( ! class_exists( 'P4_Taxonomy_Page' ) ) {
+	/**
+	 * Class P4MT_Taxonomy_Page
+	 */
+	class P4_Taxonomy_Page {
+
+		/**
+		 * Taxonomy_Page constructor.
+		 */
+		public function __construct() {
+			$this->hooks();
+		}
+
+		/**
+		 * Class hooks.
+		 */
+		private function hooks() {
+			add_action( 'category_add_form_fields',              [ $this, 'add_taxonomy_form_fields' ] );
+			add_action( 'category_edit_form_fields',             [ $this, 'add_taxonomy_form_fields' ] );
+			add_action( 'edited_category',                       [ $this, 'save_taxonomy_meta' ] );
+			add_action( 'create_category',                       [ $this, 'save_taxonomy_meta' ] );
+
+			add_filter( 'manage_edit-category_columns',          [ $this, 'edit_taxonomy_columns' ] );
+			add_filter( 'manage_category_custom_column',         [ $this, 'manage_taxonomy_custom_column' ], 10, 3 );
+			add_filter( 'manage_edit-category_sortable_columns', [ $this, 'manage_taxonomy_custom_sortable_column' ], 10, 3 );
+		}
+
+		/**
+		 * Add custom field(s) to taxonomy form.
+		 *
+		 * @param WP_Term $wp_tag The object passed to the callback when on Edit Tag page.
+		 */
+		public function add_taxonomy_form_fields( $wp_tag ) {
+
+			$args = [
+				'hierarchical'     => 1,
```

Comment:
I believe default is 1 for hierarchical.
Do we need this?

Reply:
You are right. We don't need this.